sdk/src/main/java/app/knock/client/components/models/InAppFeedFilter.kt

```diff
@@ -0,0 +1,55 @@
+package app.knock.client.components.models
+
+import android.content.Context
+import app.knock.client.R
+import app.knock.client.components.themes.EmptyFeedViewTheme
+import app.knock.client.models.feed.FeedItemScope
+
+data class InAppFeedFilter(
+    val scope: FeedItemScope,
+    val title: String = defaultTitle(scope),
+) {
+    companion object {
+        fun defaultEmptyViewConfig(context: Context, scope: FeedItemScope): EmptyFeedViewTheme {
```

Comment:
Nothing to do here yet, but wondering how we might internationalize this content? E.g. if i wanted these strings in `es` how would we do that?